lang/en.json

```diff
@@ -0,0 +1,76 @@
+  "dfreds-convenient-effects.conditionPetrified": "Petrified",
+  "dfreds-convenient-effects.conditionPoisoned": "Poisoned",
+  "dfreds-convenient-effects.conditionProne": "Prone",
+  "dfreds-convenient-effects.conditionRestrained": "Restrained",
+  "dfreds-convenient-effects.conditionStunned": "Stunned",
+  "dfreds-convenient-effects.conditionUnconscious": "Unconscious",
+  "dfreds-convenient-effects.conditionWounded": "Wounded",
+
+  "dfreds-convenient-effects.featureFavouredEnemy": "Favored Enemy",
+  "dfreds-convenient-effects.featureGreaterFavouredEnemy": "Greater Favored Enemy",
+  "dfreds-convenient-effects.featureGreatWeaponMaster": "Great Weapon Master",
+  "dfreds-convenient-effects.featureRage": "Rage",
+  "dfreds-convenient-effects.featureRagePathOfTheTotemWarrior": "Rage (Path of the Totem Warrior)",
+  "dfreds-convenient-effects.featureSharpshooter": "Sharpshooter",
+
+  "dfreds-convenient-effects.spellAid": "Aid",
+  "dfreds-convenient-effects.spellBane": "Bane",
+  "dfreds-convenient-effects.spellBarkskin": "Barkskin",
+  "dfreds-convenient-effects.spellBeaconOfHope": "Beacon of Hope",
+  "dfreds-convenient-effects.spellBless": "Bless",
+  "dfreds-convenient-effects.spellDarkvision": "Darkvision",
+  "dfreds-convenient-effects.spellEnlarge": "Enlarge",
+  "dfreds-convenient-effects.spellFly": "Fly",
+  "dfreds-convenient-effects.spellFortunesFavor": "Fortune's Favor",
+  "dfreds-convenient-effects.spellGiftOfAlacrity": "Gift of Alacrity",
+  "dfreds-convenient-effects.spellGuidance": "Guidance",
+  "dfreds-convenient-effects.spellHaste": "Haste",
+  "dfreds-convenient-effects.spellHuntersMark1h": "Hunter's Mark",
+  "dfreds-convenient-effects.spellHuntersMark8h": "Hunter's Mark (8h)",
+  "dfreds-convenient-effects.spellHuntersMark24h": "Hunter's Mark (24h)",
+  "dfreds-convenient-effects.spellLongstrider": "Longstrider",
+  "dfreds-convenient-effects.spellMageArmor": "Mage Armor",
+  "dfreds-convenient-effects.spellPassWithoutTrace": "Pass without Trace",
+  "dfreds-convenient-effects.spellReduce": "Reduce",
+  "dfreds-convenient-effects.spellShield": "Shield",
+  "dfreds-convenient-effects.spellShieldOfFaith": "Shield of Faith",
+  "dfreds-convenient-effects.spellSlow": "Slow",
+
+  "dfreds-convenient-effects.visionDim0": "Self",
+  "dfreds-convenient-effects.visionDim30": "Darkvision (30 ft)",
+  "dfreds-convenient-effects.visionDim60": "Darkvision (60 ft)",
+  "dfreds-convenient-effects.visionDim90": "Darkvision (90 ft)",
+  "dfreds-convenient-effects.visionDim120": "Darkvision (120 ft)",
+  "dfreds-convenient-effects.visionDim150": "Darkvision (150 ft)",
+  "dfreds-convenient-effects.visionDim180": "Darkvision (180 ft)",
+  "dfreds-convenient-effects.visionBright120": "Devil's Sight (Warlock)",
+
+  "dfreds-convenient-effects.lightNone": "None",
+  "dfreds-convenient-effects.lightCandle": "Candle",
+  "dfreds-convenient-effects.lightLamp": "Lamp",
+  "dfreds-convenient-effects.lightBullseye": "Lantern (Bullseye)",
```

Comment:
Not what I called any of these three lanterns in the app

Reply:
I will remove these